### PR TITLE
feat(bm-dynamic-bandwidth): Support for bandwidth while creating bare metal server

### PR DESCRIPTION
**Pull request by deepaksibm**

### CODE REVIEW

File: ibm/service/vpc/data_source_ibm_is_vpn_gateway_connection.go [hunk 1]
```diff
@@ -5,8 +5,10 @@ package vpc
 
 import (
 	"context"
+	"encoding/json"
```
Comment (SunithaGudisagarIBM1) [inline ibm/service/vpc/data_source_ibm_is_vpn_gateway_connection.go:8]: Why are these changes are part of this PR?

File: ibm/service/vpc/data_source_ibm_is_vpn_gateway_connection_local_cidrs.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
+// Copyright IBM Corp. 2024 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package vpc
+
+import (
+	"context"
+	"fmt"
+	"log"
```
Comment (SunithaGudisagarIBM1) [inline ibm/service/vpc/data_source_ibm_is_vpn_gateway_connection_local_cidrs.go:9]: Not part of bandwidth PR

File: ibm/service/vpc/resource_ibm_is_bare_metal_server_test.go [hunk 3]
```diff
@@ -46,6 +46,47 @@ ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCKVmnMOlHKcZK8tpt3MP1lqOLAcqcJzhsvJcjscgVE
...
+`)
+	sshname := fmt.Sprintf("tf-sshname-%d", acctest.RandIntRange(10, 100))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { acc.TestAccPreCheck(t) },
+		Providers:    acc.TestAccProviders,
+		CheckDestroy: testAccCheckIBMISBareMetalServerDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckIBMISBareMetalServerBandwidthConfig(vpcname, subnetname, sshname, publicKey, name, 200),
```
Comment (SunithaGudisagarIBM1) [inline ibm/service/vpc/resource_ibm_is_bare_metal_server_test.go:66]: Attach the test results as i dont think bandwidth for any profile will have number 200, 300

File: ibm/service/vpc/resource_ibm_is_share.go [hunk 4]
```diff
@@ -837,11 +837,11 @@ func resourceIbmIsShareCreate(context context.Context, d *schema.ResourceData, m
 			if replicaShareMap["name"] != nil {
 				replicaShare.Name = core.StringPtr(replicaShareMap["name"].(string))
 			}
-			if replicaShareMap["profile"] != nil {
-				replicaShare.Profile = &vpcv1.ShareProfileIdentity{
-					Name: core.StringPtr(replicaShareMap["profile"].(string)),
-				}
-			}
+			// if replicaShareMap["profile"] != nil {
```
Comment (SunithaGudisagarIBM1) [inline ibm/service/vpc/resource_ibm_is_share.go:840]: Not part of this development

File: scripts/build.sh [hunk 5]
```diff
@@ -51,13 +51,14 @@ if [[ -n "${TF_RELEASE}" ]]; then
 fi
 
 # Build!
+VERSION="1.64.0"
```
Comment (SunithaGudisagarIBM1) [inline scripts/build.sh:54]: shouldn't be part of PR

File: website/docs/r/is_bare_metal_server.markdown [hunk 6]
```diff
@@ -98,6 +98,27 @@ resource "ibm_is_bare_metal_server" "bms" {
 
 ```
 
+### Create bare metal server with bandwidth
+```terraform
+resource "ibm_is_bare_metal_server" "bms" {
+  bandwidth = 200
```
Comment (SunithaGudisagarIBM1) [inline website/docs/r/is_bare_metal_server.markdown:104]: Run terraform fmt and add workable solution

File: website/docs/r/is_vpn_gateway_connection.html.markdown [hunk 7]
```diff
@@ -92,13 +92,30 @@ Review the argument references that you can specify for your resource.
```
Comment (SunithaGudisagarIBM1) [inline website/docs/r/is_vpn_gateway_connection.html.markdown:92]: Not part of development hence not reviewing

**Review by SunithaGudisagarIBM1 — APPROVED** [review]
LGTM

**Comment by deepaksibm** [comment]
upstream merged

**Pull request closed without merging**